1.preprocessing-data/ICutils.py

```diff
@@ -0,0 +1,126 @@
+import sys
+import numpy as np
+import pathlib
+from matplotlib import pyplot as plt
+from pathlib import Path
+import os
+import skimage
+
+# explicit import to PyBaSiC due to not having package support
+sys.path.append("./PyBaSiC/")
+import pybasic
+
+def load_pybasic_data(
+    channel_path: pathlib.Path,
+    channel: str,
+    file_extension: str = ".tif",
+    verbosity: bool = True,
+):
+    """
+    Load all images from a specified directory in preparation for pybasic illum correction
+
+    Parameters:
+    channels_path (pathlib.Path): path to directory where the all images for each channel are stored
+    channel (str): The name of the channel (currently one of ["GFP", "RFP", "DAPI"])
```

Comment:
This parameter description is no longer accurate, correct?

Reply:
Correct! Fixed!